lib/src/theme/icons.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+
+/// DSFRIcons, use the remix icon icon pack
+///
+/// https://remixicon.com/
+abstract class DSFRIcons {
```

Comment:
Instead of using an `abstract class` you can use a simple `class` with a private constructor (helps to avoid warning with lint rule [avoid_classes_with_only_static_members](https://dart-lang.github.io/linter/lints/avoid_classes_with_only_static_members.html)):

```dart
@immutable
class DSFRIcons {
  const DSFRIcons._();
}
```

Reply:
if we do this we need to have a singleton. Or maybe we can define this class as a theme extension ?

Reply:
oh ok I see :)